Frontend/Controllers/Projects/AcademyPerformanceController.cs

```diff
@@ -11,23 +13,47 @@ namespace Frontend.Controllers.Projects
     public class AcademyPerformanceController : Controller
     {
         private readonly IGetInformationForProject _getInformationForProject;
+        private readonly IProjects _projectsRepository;
 
-        public AcademyPerformanceController(IGetInformationForProject getInformationForProject)
+        public AcademyPerformanceController(IGetInformationForProject getInformationForProject, IProjects projectsRepository)
         {
             _getInformationForProject = getInformationForProject;
+            _projectsRepository = projectsRepository;
         }
 
-        public async Task<IActionResult> Index(string id)
+        [HttpGet]
+        public async Task<IActionResult> Index(string id, bool addOrEditAdditionalInformation = false)
         {
             var projectInformation = await _getInformationForProject.Execute(id);
 
             var model = new AcademyPerformanceViewModel
             {
                 Project = projectInformation.Project,
-                OutgoingAcademy = projectInformation.OutgoingAcademy
+                OutgoingAcademy = projectInformation.OutgoingAcademy,
+                AdditionalInformationModel = new AdditionalInformationViewModel
+                {
+                    AdditionalInformation = projectInformation.Project.AcademyPerformanceAdditionalInformation,
+                    HintText = "This information will populate into your HTB template under the school performance (Ofsted information) section.",
+                    Urn = projectInformation.Project.Urn,
+                    AddOrEditAdditionalInformation = addOrEditAdditionalInformation
+                }
             };
 
             return View(model);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> Index(string id, string additionalInformation)
+        {
+            var model = await _projectsRepository.GetByUrn(id);
+
+            model.Result.AcademyPerformanceAdditionalInformation = additionalInformation;
+            await _projectsRepository.Update(model.Result);
```

Comment:
I'd add a test for each controller also that makes sure that update gets called with the updated project information (Using Moq Verify)